src/objective/objective_function.cpp

```diff
@@ -8,9 +8,14 @@ namespace LightGBM {
 
 ObjectiveFunction* ObjectiveFunction::CreateObjectiveFunction(const std::string& type, const ObjectiveConfig& config) {
   if (type == std::string("regression")) {
+    // deprecated
     return new RegressionL2loss(config);
+  } else if (type == std::string("regression_l2") || type == std::string("mean_squared_error") || type == std::string("mse")) {
+    return new RegressionL2loss(config);
```

Comment:
I'm sorry, I don't understand what you mean 😢
This PR will be merged after #175